src/web/components/ClientSideTokenGeneration/CstgDomainsTable.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Story } from '@storybook/react';
+
+import { CstgDomainsTable, CstgDomainsTableProps } from './CstgDomainsTable';
+
+export default {
+  title: 'CSTG/CstgDomainsTable',
+  component: CstgDomainsTable,
+} as Meta<typeof CstgDomainsTable>;
+
+const Template: Story<CstgDomainsTableProps> = (args) => <CstgDomainsTable {...args} />;
```

Comment:
Using `Story<Props>` instead of `ComponentStory<typeof Component>` Give us type check for args:

Since v7 ComponentStory and ComponentMeta are deprecated, maybe we should upgrade to v7
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#componentstory-componentstoryobj-componentstoryfn-and-componentmeta-types-are-deprecated